lib/puppet/provider/razor_rest.rb

```diff
@@ -33,11 +33,38 @@ def self.get_rest_info
     else
       port = 8150
     end
+     if yamldata.include?('http_method')
+      http = yamldata['http_method']
+    else
+      http = 'http'
+    end
+
+    if yamldata.include?('client_cert')
+      client_cert = yamldata['client_cert']
+    else
+      client_cert = "/etc/puppetlabs/puppet/ssl/certs/#{hostname}.pem"
```

Comment:
I think these paths should default to `nil`. The installation module should be responsible for helping this module locate these files. I suggest adding these as module parameters in `init.pp`.

Reply:
Mostly just following the existing code logic to maintain consistency.